src/ui/components/Select/styles.scss

```diff
@@ -0,0 +1,27 @@
+@import "~photon-colors/colors";
+
+@import "~core/css/inc/mixins";
+@import "~ui/css/vars";
+
+.Select {
+  @include padding-end(22px);
+  @include padding-start(5px);
```

Comment:
I wonder if all these styles need updating? It can be done later if you like but seeing a lot of multiples of 5 in here when often the new styles are multiples of 6. If it's easy and can be folded into this patch then 👍, otherwise no worries.

Reply:
Good catch, thanks